cypress/support/commands.ts

```diff
@@ -50,6 +50,40 @@ Cypress.Commands.add("addMachine", (hostname = generateName()) => {
   });
 });
 
+Cypress.Commands.add("deleteMachine", (hostname: string) => {
+  cy.visit(generateMAASURL("/machines"));
+  cy.findByRole("combobox", { name: "Group by" }).select("No grouping");
+  cy.findByRole("searchbox").type(hostname);
+  cy.findByText(/1 machine available/).should("exist");
+  cy.findByRole("grid", { name: "Machines" }).within(() =>
+    // eslint-disable-next-line cypress/no-force
+    cy
+      .findByRole("checkbox", { name: new RegExp(hostname) })
+      .click({ force: true })
```

Comment:
Is there a different behavior if this isn't forced?

Reply:
Yes, this test would fail as the native HTML checkbox is not visible (we're hiding the original checkbox via CSS to display custom Vanilla framework styles)